toqito/channels/realignment.py

```diff
@@ -1,10 +1,29 @@
 """Generates the realignment channel of a matrix."""
 
 import numpy as np
+from collections.abc import Iterable
 
 from toqito.matrix_ops import partial_transpose
 from toqito.perms import swap
 
+def _validate_dim_param(dim):
+    """Validate dim parameter: must be int > 0 or iterable of ints > 0."""
+    # Accept None (preserve current behaviour)
+    if dim is None:
+        return
+    # Check iterable of ints (exclude str/bytes)
```

Comment:
It seems weird to explicitly exclude these types. Couldn't we instead have some type of "else" catch all if all of these `if` statements fail?